core/src/main/java/com/tweet/core/repository/TweetRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.tweet.core.repository;
+
+import com.tweet.core.model.Tweet;
+import org.springframework.data.jpa.repository.JpaRepository;
+
```

Comment:
Please add @Repository annotation so spring can find this bean.
Beyond that, it activates persistence exception translation for all beans annotated with @Repository, to let exceptions being thrown by the JPA persistence providers be converted into Spring’s DataAccessException hierarchy.